External/Plugins/ASCompletion/Completion/ASGenerator.cs

```diff
@@ -2793,22 +2792,10 @@ public static void GenerateExtractVariable(ScintillaControl Sci, string NewName)
                         }
                     }
 
-                    int style = Sci.StyleAt(funcBodyStart + lastPos) & stylemask;
-                    if (ASComplete.IsCommentStyle(style))
-                    {
-                        continue;
-                    }
-                    else if ((isExprInDoubleQuotes && currentMethodBody[lastPos] == '"' && currentMethodBody[lastPos + expression.Length - 1] == '"')
-                        || (isExprInSingleQuotes && currentMethodBody[lastPos] == '\'' && currentMethodBody[lastPos + expression.Length - 1] == '\''))
-                    {
-
-                    }
-                    else if (!ASComplete.IsTextStyle(style))
-                    {
-                        continue;
-                    }
-
-                    Sci.SetSel(funcBodyStart + lastPos, funcBodyStart + lastPos + expression.Length);
+                    var pos = funcBodyStart + lastPos;
+                    int style = Sci.StyleAt(pos) & stylemask;
```

Comment:
@elsassph, code:
```
int stylemask = (1 << Sci.StyleBits) - 1;
...
Sci.StyleAt(pos) & stylemask
```
equals to
```
Sci.BaseStyleAt(pos)
```
I have to replace "Sci.StyleAt(pos) & stylemask" to "Sci.BaseStyleAt(pos)"?